starter_code/public/stylesheets/style.css

```diff
@@ -0,0 +1,67 @@
+body {
+  padding: 0;
+  margin: 0;
+  font: 14px "Lucida Grande", Helvetica, Arial, sans-serif; }
+
+a {
+  color: #00b7ff; }
+
+.homepage-container {
+  background-image: url(/images/cinema.jpg);
+  background-size: 100%;
+  background-repeat: no-repeat;
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-end; }
+
+.headline-container {
+  padding-right: 300px; }
+
+.home-headline {
+  color: white;
+  font-size: 66px;
+  width: 200px;
```

Comment:
Keep in mind that this (and all the ones underneath) is not responsive. I personally prefer to use vw instead of px.
e.g: width: 15vw; min-width: 100px